build-scripts/webpack.js

```diff
@@ -75,6 +76,7 @@ const createWebpackConfig = ({
       chunkIds: isProdBuild && !isStatsBuild ? "deterministic" : "named",
     },
     plugins: [
+      new CompressionWebpackPlugin(),
```

Comment:
this adds it to every build (stats, prod, dev). And since we already compress as part of our prod step outside webpack, we're now compressing production builds twice.